special-interest-groups/sig-dashboard/roles-and-organization-management.md

```diff
@@ -0,0 +1,75 @@
+# Dashboard SIG Roles and Organization Management
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
+
+### Join
+
+There is no restrictions to join the discussion of Dashboard SIG. See [README](./README.md) for where we discuss.
+
+However you will be only listed in our [member list] if you have **Active Contributor** role or above.
+
+### Promotion
+
+#### Concept: PR Point
+
+Most of the promotion rules below use "PR Point" to calculate the contribution, which is calculated as:
+
+- 1 easy level task = 1 point
+- 1 medium level task = 5 point
+- 1 hard level task = 15 point
+
+#### Be Promoted to [`Contributor`](../../architecture/README.md#contributor)
+
+Everyone that successfully contributes commits to the [TiDB Dashboard repository](https://github.com/pingcap-incubator/tidb-dashboard) is a contributor.
+
+#### Be Promoted to [`Active Contributor`](../../architecture/README.md#active-contributor)
+
+Please open a Pull Request to list yourself in our [member list] if you meet the following requirements and want to have an Active Contributor role:
+
+- PR point >= 5 in recent 1 years.
+
+#### Be promoted to [`Reviewer`](../../architecture/README.md#reviewer)
+
+Tech Leaders can nominate promoting a member to Reviewer.
+
+There is no explicit promotion rules, but the following basic requirements must be met:
+
+- Nominated by a Tech Leader.
+- Already an Active Contributor.
+- Familiar with the code base.
+- Contributions that prove the reviewing capability: PR point >= 10.
+
+#### Be promoted to [`Committer`](../../architecture/README.md#committer)
+
+Tech Leaders can nominate promoting a member to Committer.
+
+There is no explicit promotion rules, but the following basic requirements must be met:
+
+- Nominated by a Tech Leader.
+- Approved by 2 maintainers from TiDB Dashboard, TiDB or TiKV.
+- Already an Active Contributor (Reviewer is not a prerequisite).
+- Exhibited good technical judgment.
+- Notable contributions: PR point (**excluding** easy level) >= 20.
+
+## Quit and Demotion
+
+### Active Contributor
```

Comment:
we also need to describe these rules for `reviewer` and `committer`.

Reply:
There is intentionally no demotion for Reviewer and Committer.

Reply:
we should have one.

Reply:
I think this is a SIG specific.